docs/en/1_exist_data_model.md

```diff
@@ -499,9 +499,6 @@ pipelines = [
 MMDetection also provides out-of-the-box tools for training detection models.
 This section will show how to train _predefined_ models (under [configs](https://github.com/open-mmlab/mmdetection/tree/master/configs)) on standard datasets i.e. COCO.
 
-**Important**: The default learning rate in config files is for 8 GPUs and 2 img/gpu (batch size = 8\*2 = 16).
```

Comment:
We do not need to clean the doc. We should update it and tell users we already support auto-scaling LR.

Reply:
Done , I added it under the `Learning rate automatically scale`. Also I added instructions on how to distinguish whether it is the default batch size and how to modify the default batch size.